src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,14 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+/**
+ * Represents a contact in contact list
```

Comment:
Does not have to be too specific on where the class is used (i.e. `in the contact list`). Make it more general. `Contact` does not always have to be bounded by a list.